### ISSUE TITLE
`editor.renderRichScreenReaderContent` description should be a markdownDescription

### ISSUE THREAD

**Issue by connor4312** [microsoft/vscode#258661]
Testing #257834

Notice the backticks are not interpreted

<img width="616" height="67" alt="Image" src="https://github.com/user-attachments/assets/f4b20f06-5eb6-45e6-98f0-6641cf750dfe" />

**Comment by aiday-mar** [comment]
To verify:

- Check that the `editor.renderRichScreenReaderContent` setting has a link to the `editor.editContext` in the description.
